app/packages/core/src/components/Sidebar/Entries/Draggable.tsx

```diff
@@ -19,8 +19,10 @@ const Draggable: React.FC<
   const theme = useTheme();
   const [hovering, setHovering] = useState(false);
   const [dragging, setDragging] = useState(false);
-  const isReadOnly = useRecoilValue(readOnly);
-  const isFieldVisibilityApplied = useRecoilValue(isFieldVisibilityActive);
+  const isSnapShot = useRecoilValue(fos.readOnly);
+  const canAddSidebarGroup = useRecoilValue(fos.canAddSidebarGroup);
```

Comment:
Should we use a different one here (i.e. `canUpdateSidebarOrder`) or rename it to something more general `canEditSidebar`?

Reply:
How about`canModifySidebarGroups`?